licenses/APACHE2.rst

```diff
@@ -0,0 +1,84 @@
+Copyright 2018, Astropy Developers
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
@bsipocz @Cadair - wait, why do we have the apache license?

Reply:
The rendered version should only have the BSD 3-clause, right?

Reply:
We have templated in *all* the licences so people who clone master can pick one if they wish.

Reply:
Ok sounds good